gabrielaEnache/src/main/java/collections/store/Item.java

```diff
@@ -0,0 +1,42 @@
+package collections.store;
+
+public class Item {
+    private static long id = 0;
+    private String title;
+    private Type type;
+
+    public Item(String title, Type type) {
+        id++;
```

Comment:
id should not be static because this is an instance (object ) related attribute. In case you leave it as it is, the id variable will be equal to total number of items that you have created. I would suggest the following solution:
1. Create an private static long idCounter variable.
2. Chenge the id property to private long id
3. within constructor provide correct value to newly created item : this.id = idCounter++

Reply:
Fixed